metale_amorficzne/pages/02_Data.md

```diff
@@ -0,0 +1,21 @@
+# Data
+
+## Materials and research methods used for this study (Data)
+
+- Zirconium Copper (ZrCu) and Zirconium Titanium (ZrTi) alloys,
+- nanoindentation of specifically prepared, polished metal samples in the form
+  of thin coins. Indentation was performed on each sample in a matrix of 15×15
+  or 10×10 points in distances ranging from 5 to 20 um, force applied ranging
+  from 25mN to 200mN.
+- data output from nanoindentation machine *CSM Instruments NHT2* comes as .TXT
+  file and contains calculations of 21 parameters for each indentation point,
+---
```

Comment:
After splitting files, I think that you don't need to use the horizontal split lines.

Reply:
ok